webapi/src/main/scala/org/knora/webapi/slice/admin/api/GroupsEndpoints.scala

```diff
@@ -26,21 +36,43 @@ final case class GroupsEndpoints(baseEndpoints: BaseEndpoints) {
     .description("Returns all groups.")
 
   val getGroupByIri = baseEndpoints.publicEndpoint.get
-    .in(base / groupIri)
+    .in(base / groupIriPathVar)
     .out(sprayJsonBody[GroupGetResponseADM])
     .description("Returns a single group identified by IRI.")
 
   val getGroupMembers = baseEndpoints.securedEndpoint.get
-    .in(base / groupIri / "members")
+    .in(base / groupIriPathVar / "members")
     .out(sprayJsonBody[GroupMembersGetResponseADM])
     .description("Returns all members of a single group.")
 
-  private val securedEndpoins = Seq(getGroupMembers).map(_.endpoint)
+  val postGroup = baseEndpoints.securedEndpoint.post
+    .in(base)
+    .in(zioJsonBody[GroupCreateRequest])
+    .out(sprayJsonBody[GroupGetResponseADM])
+    .description("Creates a new group.")
+
+  private val securedEndpoins = Seq(getGroupMembers, postGroup).map(_.endpoint)
 
   val endpoints: Seq[AnyEndpoint] = (Seq(getGroups, getGroupByIri) ++ securedEndpoins)
     .map(_.tag("Admin Groups"))
 }
 
+object GroupsRequests {
+  import org.knora.webapi.slice.admin.api.Codecs.ZioJsonCodec.*
+  final case class GroupCreateRequest(
+    id: Option[GroupIri] = None,
+    name: GroupName,
+    descriptions: GroupDescriptions,
+    project: ProjectIri,
+    status: GroupStatus,
+    selfjoin: GroupSelfJoin
+  )
+
+  object GroupCreateRequest {
+    implicit val jsonCodec: JsonCodec[GroupCreateRequest] = DeriveJsonCodec.gen[GroupCreateRequest]
+  }
+}
```

Comment:
Maybe I'm missing context here and this is needed - but does it really make sense to have a dedicated object just to hold a case class and an object?

Reply:
In the other endpoints we have at some point had more than one and grouping all request bodies in an object seems reasonable to me.

Reply:
@BalduinLandolt there should be more requests coming into this object, otherwise I agree the parent object could be removed.

Reply:
makes sense, thanks!